src/core.cpp

```diff
@@ -53,7 +94,7 @@ void CVC::EvaluateQueuedActions() {
 
 void CVC::ChooseActions() {
     //go through list of all characters
-    for(auto const& character: this->characters_) {
+    for(auto&& character: this->characters_) {
```

Comment:
...this seems wrong to me

Reply:
well, you aren't moving from character, so I think auto& character is clearer